runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkJobServerDriver.java

```diff
@@ -94,6 +95,18 @@ String getFlinkMasterUrl() {
     Long getSdkWorkerParallelism() {
       return this.sdkWorkerParallelism;
     }
+
+    @Option(
+      name = "--flink-conf-dir",
+      usage =
+          "Directory containing Flink YAML configuration files. These properties will be set to all jobs submitted to Flink."
```

Comment:
Mention here that it overrides `FLINK_CONF_DIR` and also confirm that if this option isn't used `FLINK_CONF_DIR` remains in effect.

Reply:
Updated